### PR TITLE
Declare variables and functions

### PR DESCRIPTION
**Pull request by zachswift615**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -1,162 +1,247 @@
...
+ example:
+ note: this example shows the result after completing steps 3-8
+ */
+bool rentACar(int rentalDuration, int carType = 0)  //function declaration with random number of arguments, arbitrary number of arguments have default value
+{
+    ignoreUnused(rentalDuration, carType); //passing each function parameter to the ignoreUnused() function
+    return 
+    {
+
+    }; //if your function returns something other than void, add 'return {};' at the end of it.
```
Comment (matkatmusic) [inline main.cpp:100]: write `return {};` not 
```
return 
{

};
```
- [ ] revise

File: main.cpp [hunk 2]
```diff
@@ -1,162 +1,247 @@
...
+
+/*
+ 1)
+ */
+int findMusicInterval(char note1, char note2)
+{
+    ignoreUnused(note1, note2);
+    return {
+
+    };
```
Comment (matkatmusic) [inline main.cpp:111]: write `return {};` not 
```
return {

};
```
- [ ] revise

File: main.cpp [hunk 3]
```diff
@@ -1,162 +1,247 @@
...
+ 4)
+ */
+float cookRawParamValue(float rawValue)
+{
+    ignoreUnused(rawValue);
+    return 
+    {
+
+    };
+}
```
Comment (matkatmusic) [inline main.cpp:141]: write `return {};` not 
```
return 
{

};
```
- [ ] revise

File: main.cpp [hunk 4]
```diff
@@ -1,162 +1,247 @@
...
+ 7)
+ */
+int findNextZeroCrossing(int sample1, int sample2, int sample3)
+{
+    ignoreUnused(sample1, sample2, sample3);
+    return 
+    {
+      
+    };
+}
```
Comment (matkatmusic) [inline main.cpp:170]: write `return {};` not 
```
return 
{

};
```
- [ ] revise

File: main.cpp [hunk 5]
```diff
@@ -1,162 +1,247 @@
...
+    pin the pull request link and this repl.it link to our DM thread in a single message.
+
+ send me a DM to review your pull request when the project is ready for review.
+
+ Wait for my code review.
+ */
+
+int main() {
+    //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
+    auto carRented = rentACar(6, 2);
```
Comment (matkatmusic) [inline main.cpp:212]: curly braces go on their own line
- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 5 above)
Comment (zachswift615) [reply main.cpp]: With the exception of `return {};` I'm gathering. 😆   
I updated this

**Pull request merged**